### DISCUSSION TITLE
Extracting Data From Cif File

### DISCUSSION THREAD

**Discussion by SamWright822** [project-gemmi/gemmi#190] [Q&A] [answered]
I have read through the documentation, and I want to use gemmi to read and parse a CIF file. I'm unsure how to access certain information in the document once read. For instance, if I want to save the Hall Symbols, H-M symbols, or other information. I am unsure from the documentation how to extract this information. 
Thanks!

**Comment by wojdyr** [comment, marked as answer]
```py
from gemmi import cif
doc = cif.read(path)
block = doc[0]
value_possibly_with_quotes = block.find_value('_symmetry_space_group_name_Hall')
value = cif.as_string(value_possibly_with_quotes)
```

**Reply by SamWright822** [reply]
Thank you!
